lib/chef_apply/config.rb

```diff
@@ -111,6 +111,8 @@ def reset
     config_context :log do
       default(:level, "warn")
       default(:location, File.join(WS_BASE_PATH, "logs/default.log"))
+      # set the log level for the target host's chef-client run
+      default(:target_level, nil)
```

Comment:
A slight change in behaviour from the original PR state: this is now intended to allow leaving this parameter default (nil) and avoid including an explicit `log_level` in `workstation.rb`. This allows the target host's chef-client process to default to whatever is the default.

It still allows setting a value on the chef-apply side, and this value will be applied to `workstation.rb` as intended.

Tests amended to this effect.